### PR TITLE
MAPI-1421 - applied MacWire to remaining services

### PR DESCRIPTION
**Pull request by agulowaty**
* introduced MacWire to registration service
* MacWired report service

cc @davidfurey @alexduf

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -60,8 +60,8 @@ lazy val backup = project
     version := "1.0-SNAPSHOT"
   )
 
-lazy val registration = project.
-  dependsOn(common)
+lazy val registration = project
+  .dependsOn(common % "test->test;compile->compile")
```
Comment (alexduf) [inline build.sbt]: Do you have a new dependency between common/test and registration/test?

File: registration/app/registration/RegistrationApplicationLoader.scala [hunk 2]
```diff
@@ -0,0 +1,107 @@
+package registration
+
+import _root_.controllers.Assets
```
Comment (alexduf) [inline registration/app/registration/RegistrationApplicationLoader.scala]: is the _root_ necessary here?

File: build.sbt (hunk 1 above)
Comment (agulowaty) [inline build.sbt]: This a way of sharing `common/src/test/scala/application/WithPlayApp.scala` trait between several projects (test-scoped)

File: registration/app/registration/RegistrationApplicationLoader.scala (hunk 2 above)
Comment (agulowaty) [inline registration/app/registration/RegistrationApplicationLoader.scala]:
```
[error] /Users/agulowaty/guardian/mobile-n10n/registration/app/registration/RegistrationApplicationLoader.scala:3: object Assets is not a member of package registration.controllers
[error] import controllers.Assets
```

**Comment by alexduf** [comment]
Nothing blocking, looks good :+1:

File: registration/app/registration/RegistrationApplicationLoader.scala (hunk 2 above)
Comment (alexduf) [inline registration/app/registration/RegistrationApplicationLoader.scala]: I'll take that as a yes :)

**Pull request merged**